server/container/testing/pom.xml

```diff
@@ -32,6 +32,10 @@
 
     <name>Apache James :: Server :: Deployment Testing</name>
 
+    <properties>
+        <testing-deployment.includes>empty</testing-deployment.includes>
```

Comment:
> please use a junit tag instead ( https://junit.org/junit5/docs/current/user-guide/#writing-tests-tagging-and-filtering )

With tests written using JUNIT 4 ?

Reply:
I think that we should be able to write tests in junit5

Reply:
Can you explain me the added value here using junit tags instead of profile?

Reply:
> Can you explain me the added value here using junit tags instead of profile?

Profiles serve a lot of different purposes. It's really easy to mess with them and have thing to never run at all. We did a lot of that in the past.

Tag's intent is very clear : it's here to group tests so that you can run some of them without the others. So I would say it's just a matter of using the right abstraction so that people don't get fooled by tricks they don't understand or know about.

I agree that the result in term of execution is exactly the same. The result for the developer is not though.

Reply:
ok for that, but then you need a specific maven profile to activate or deactivate some specific tagged tests, no?

Reply:
I'd rather put an explicit list of Tags (groups in surefire terms) on the command line

Reply:
ok, finally got it, you have to use -Dgroups. :+1: then